conf/bk_server.conf

```diff
@@ -749,6 +749,9 @@ gcEntryLogMetadataCacheEnabled=false
 # By default it will be allocated to 25% of the available direct memory
 # dbStorage_readAheadCacheMaxSizeMb=
 
+# How many entries' bytes to pre-fill in cache after a read cache miss
+# dbStorage_readAheadCacheBatchBytesSize=-1
```

Comment:
How about change the default value to 0?  Because 0 will also disable this feature.

And we'd better add a comments like this:
`Default is 0. 0 or less disables this feature`

Reply:
> How about change the default value to 0? Because 0 will also disable this feature.
> 
> And we'd better add a comments like this: `Default is 0. 0 or less disables this feature`

updated,thanks